packager/VisibilityChecker.cc

```diff
@@ -821,6 +835,152 @@ class VisibilityCheckerPass final {
         return files;
     }
 };
+
+bool ownerAlreadyExported(const core::GlobalState &gs, vector<core::SymbolRef> &alreadyExported,
```

Comment:
Suggestion for another name: `ownerWillBeExported`

On first read, I thought that this function would be looking at the exported bit, but in fact, it's looking in the toExport list.